san/v2_metrics_list.py

```diff
@@ -0,0 +1,62 @@
+V2_METRIC_QUERIES = [
```

Comment:
Can we replace this by an API call to https://api.santiment.net/graphiql?variables=&query=%7B%0A%20%20getAvailableMetrics%0A%7D%0A?

Reply:
Yes it can, I will only implement the getAvailableMetrics and call it, but is it better than just having the file? Another thing, the README points exactly to that file, where the available metrics can be seen.

Reply:
The current list will be outdated when I merge https://github.com/santiment/sanbase2/pull/1398

We can have README data that is not connected to the list here :?

Reply:
You're right, that it will be outdated.